src/test/java/ut/org/catrobat/jira/timesheet/services/impl/TimesheetServiceImplTest.java

```diff
@@ -225,7 +223,7 @@ public void testEditTimesheets() throws ServiceException {
         Timesheet.State expectedState = Timesheet.State.ACTIVE;
 
         Timesheet timesheet = ao.create(Timesheet.class, new DBParam("USER_KEY", userKey));
-        timesheet.setHoursPracticeCompleted(originalHoursPracticeCompleted);
+        timesheet.setHoursCompleted(originalHoursPracticeCompleted);
```

Comment:
Please remove this line including the deprecated `originalHoursPracticeCompleted` variable.